hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/local/LocalRegionCache.java

```diff
@@ -363,6 +371,16 @@ private Duration resolveTTL() {
           : evictionConfig.getTimeToLive();
     }
 
+    private MessageListener<Object> ignoreMessagesFromLocalMember(final MessageListener<Object> delegate) {
```

Comment:
Why do we need this additional function? Isn't it fine to modify `createMessageListener` only? They both are identical anyway.

Reply:
Good question. Since the scope of `createMessageListener` is protected there is a small chance that some subclass may (intentionally or not) bypass the change introduced in this PR by overriding that method should I put the logic there. I just want to be on the safe side here by having the logic in this `private` method and use it to call the `protected` one.

Reply:
There is no other usage of it from subclasses. Let's make it private and make sure we're safe with the listener.